pom.xml

```diff
@@ -24,7 +24,7 @@
     <properties>
         <opencga.version>0.5.2</opencga.version>
         <compileSource>1.8</compileSource>
-        <mongo-java-driver.version>2.14.2</mongo-java-driver.version>
+        <mongo-java-driver.version>3.2.2</mongo-java-driver.version>
```

Comment:
Version number no longer necessary, it is inherited from `spring-boot-starter-data-mongodb`.